### ISSUE TITLE
Excluding other extensions when an extension is present

### ISSUE THREAD

**Issue by zhouyx** [ampproject/amphtml#21723]
Right now we have some extensions that don't work well together. For example when `<amp-skimlink>` and `<amp-link-rewrite>` works together, the ladder component can affect what the previous component does. 
It would be useful to have a feature that only allow one component on a page among a list of mutual exclusive extensions. 

@ampproject/wg-caching

**Comment by honeybadgerdontcare** [comment]
When discussing this I was thinking of `requires_extension` and that perhaps we should also have a `excludes_extension` feature. This is to be more explicit than `satisfies`, `requires` and `excludes`.

**Comment by zhouyx** [comment]
I agree. `excludes_extensions` is also much easier to understand : )

**Comment by jpettitt** [comment]
+1 for `excludes_extensions`

`amp-access` and `amp-subscriptions` are mutually exclusive.

**Comment by Gregable** [comment]
@zhouyx I think this got forgotten. Is this still important to you?

**Comment by zhouyx** [comment]
I think the issue still exist. But I think not urgent because there's no incentive for one to include two components either from different service provider (amp-skimlink vs amp-smartlink) or designed for the same purpose (amp-access vs amp-subscriptions).

**Comment by Gregable** [comment]
Thanks for the update. I'll keep it around, but let it remain at P3 for now. Would be nice for a fixit.

**Comment by honeybadgerdontcare** [comment]
Is anyone still interested in this being done?
